### PR TITLE
Restructure SDF/URDF files.

### PR DESCRIPTION
**Pull request by varunagrawal**
Moved the models to a common `models` directory so we can easily add more directories (e.g. from the DARPA subt challenge) without cluttering the root directory.

### CODE REVIEW

**Comment by varunagrawal** [comment]
ping @gchenfc

File: python/notebooks/rendering_urdf.ipynb [hunk 1]
```diff
@@ -49,62 +47,62 @@
...
+   ],
+   "outputs": [],
+   "metadata": {}
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
-   "outputs": [],
    "source": [
```
Comment (gchenfc) [inline python/notebooks/rendering_urdf.ipynb:57]: not a request for changing anything, but just generally I wonder if this is a jupyter notebook version thing or if generally there is a better way to version control notebooks?

**Review by gchenfc — APPROVED** [review]
LGTM - sorry for late review on this very quick review PR

File: python/notebooks/rendering_urdf.ipynb (hunk 1 above)
Comment (varunagrawal) [reply python/notebooks/rendering_urdf.ipynb:57]: There actually is a way, but I don't have the time to figure it out right now.

Here's some help: https://zhauniarovich.com/post/2020/2020-06-clearing-jupyter-output/

**Pull request merged**